suse_migration_services/units/reboot.py

```diff
@@ -57,3 +54,19 @@ def main():
         Command.run(
             ['reboot', '-f']
         )
+
+
+def is_debug_mode():
+    """
+    Returns True or False depending on debug mode
+    """
+    migration_config_file = os.sep.join(
+        ['/etc', os.path.basename(Defaults.get_system_migration_config_file())]
+    )
+    config = _read_system_migration_config(migration_config_file)
+    return config['debug']
+
+
+def _read_system_migration_config(config_file_path):
```

Comment:
I think the yaml.load() doesn't need to be put into an extra private method. I would just add that code in the is_debug_mode() code as it's used only once and basically fits into the code flow. I leave this part up to you though

Reply:
I have removed the method and added that code inside ```is_debug_mode()```.
I was following the same approach as [get_migrate_product](https://github.com/SUSE/suse-migration-services/blob/fd890a2cf4d8440e98da92823b4e64ff2f51728b/suse_migration_services/units/migrate.py#L90).